### PR TITLE
Feature name and pic

### PR DESCRIPTION
**Pull request by ryankawahara**
## Description
Added the user's name and profile picture to the login state. Sometimes the code has a hard time accessing the profile picture so there is a placeholder image to handle that error.

## Relevant Issue
Previously it was difficult to tell which user was logged in.

## Tests
n/a

### CODE REVIEW

**Review by Jyotsnaiyer — CHANGES_REQUESTED** [review]
Ryan please can you put up the latest update for this

**Review by ryankawahara — COMMENTED** [review]
Made requested changes

File: source/frontend/src/components/Navbar.jsx [hunk 1]
```diff
@@ -5,49 +5,63 @@ import PropTypes from 'prop-types'
...
     };
 
     const onSuccessLogout = (res) => {
         console.log("Log out successfull");
-        console.log("badaboom");
         setIsUserLoggedIn(false);  // Set login status to true on successful login
 
     };
-    return (
-        <div className='navbar p-8 border mb-12 flex justify-between items-center'> {/* Flexbox container */}
```
Comment (Jyotsnaiyer) [inline source/frontend/src/components/Navbar.jsx:31]: this part is mentioned twice

File: source/frontend/src/components/Navbar.jsx [hunk 2]
```diff
@@ -5,49 +5,63 @@ import PropTypes from 'prop-types'
...
         console.log("LOGIN SUCCESS! Current user :", res.profileObj);
-        console.log("badaboom");
         setIsUserLoggedIn(true);  // Set login status to true on successful login
-
+        console.log(res.profileObj.imageUrl);
+        setProfileImageUrl(res.profileObj.imageUrl);
+        setUsername(res.profileObj.name);
     };
 
     const onSuccessLogout = (res) => {
```
Comment (Jyotsnaiyer) [inline source/frontend/src/components/Navbar.jsx:30]: The onSuccesLogout  lets us know when the user has logged out but it does not clear out the 'profileimageurl' and 'username' state which could eventually be leaked.
'Profileimageurl' and 'username' should be reverted back to original state
Below code could be added
const onSuccessLogout = (res) => {
    console.log("Logout successful");
    setIsUserLoggedIn(false);  
    setProfileImageUrl(null); 
    setUsername(null);         
};

**Review by Jyotsnaiyer — CHANGES_REQUESTED** [review]
Ryan, please can you address the 2 comments

File: source/frontend/src/components/Navbar.jsx (hunk 1 above)
Comment (ryankawahara) [reply source/frontend/src/components/Navbar.jsx:31]: where do you see the other part @Jyotsnaiyer ? I only see it there

Comment (Jyotsnaiyer) [reply source/frontend/src/components/Navbar.jsx:31]: Line 43 to line 50

**Review by Jyotsnaiyer — APPROVED** [review]
Navbar.jsx file edited and suggested code also added

**Pull request merged**